Add support phpStorm 2023.3 EAP

CrazyBoy49z opened the pull request:
https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-extension.html#intellijPlatform-pluginConfiguration-ideaVersion

#1  Maybe this will solve the problem

CrazyBoy49z committed:
Add support phpStorm 2023.3 EAP

https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-extension.html#intellijPlatform-pluginConfiguration-ideaVersion

aSemy commented on build.gradle.kts:
```diff
@@ -45,6 +45,7 @@ intellijPlatform {
     version = provider { project.version.toString() }
     ideaVersion {
       sinceBuild = "242"
+      untilBuild = "242.*"
     }
```
https://github.com/aSemy/ij-problem-ignorer/blob/v0.0.4/build.gradle.kts#L44-L47

This is the same config I had in v0.0.4 though :(

CrazyBoy49z replied:
I’ll be watching over the weekend, can you tell me how I can build the plugin locally?

aSemy replied:
This repo uses the official IJ Gradle Plugin, so all the info should be in the docs :)

https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html

CrazyBoy49z replied:
> This repo uses the official IJ Gradle Plugin, so all the info should be in the docs :)
> 
> https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html

FIxed

CrazyBoy49z replied:
<img width="800" alt="image" src="https://github.com/user-attachments/assets/f84812a6-374f-484b-a44c-d5516b8007b5">

CrazyBoy49z committed:
Add JetBrains repositories to Gradle build files

Integrated JetBrains snapshot and release repositories into main and buildSrc Gradle files. Updated the IntelliJ platform plugin version compatibility from "242.*" to "243.*".

aSemy commented:
See https://github.com/aSemy/ij-problem-ignorer/issues/1#issuecomment-2383717404
